tools/release/publish_release.sh

```
@@ -774,7 +823,21 @@ fi
 # ======================================
 
-if [ "$cherry_pick_resolved" = "false" ]; then
+if [ "$continue_from_changelogs" = "true" ]; then
+    # Resume an interrupted release: the rc branch was already created and pushed,
+    # but the changelog step failed. Skip cherry-picking, check out the existing rc
+    # branch, wipe the changelog branches, and fall through to the changelog / QA /
+    # announce steps below.
+    if [ "$dry_run" = "false" ]; then
+        ask "About to switch to '$target_branch' to resume. Make sure your working tree is clean first (uncommitted changes from a previous perl/version run can block the checkout). Continue? [y/N] "
+        git fetch
+        git checkout $target_branch
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)